libs/sdk-ui-dashboard/src/model/events/drill.ts

```diff
@@ -551,6 +551,10 @@ export interface DashboardDrillToCustomUrlResolved extends IDashboardEvent {
          * Resolved custom url.
          */
         readonly url: string;
+        /**
+         * Information about filters used on the dashboard.
+         */
+        readonly filtering: FiltersInfo;
```

Comment:
Nitpick: can this be just `filters`? I'd prefer if we avoided the unnecessary -ing suffix we use sometimes.
Same in the other event in the file.

Reply:
The thing is that it consists of `filters` and `resolvedFilterValues`, so we wanted to avoid repeating the `filters` keyword.